### PR TITLE
Improve subgraph documentation.

### PR DESCRIPTION
**Pull request by iamashwin99**
- Fix some typos
- Add a link to the graph playground

### CODE REVIEW

File: docs/web/subgraph.mdx [hunk 1]
```diff
@@ -24,11 +24,13 @@ The schema for the ENS subgraph is defined in [/schema.graphql](https://github.c
 
 ## Use Cases
 
-There are certain use cases where the graph is better for querying ENS specific information then through the resolution process.
+There are certain use cases where the graph is better for querying ENS specific information than through the resolution process.
 One of such use-cases is querying which NFT names are owned by a specific address.
 
 ## Example Queries
 
+One can explore the following examples interactively via the [Graph Explorer Playground](https://thegraph.com/explorer/subgraphs/EjtE3sBkYYAwr45BASiFp8cSZEvd1VHTzzYFvJwQUuJx?view=Playground&chain=mainnet)
```
Comment (gskril) [inline docs/web/subgraph.mdx:32]:
```suggestion
One can explore the following examples interactively via the [Graph Explorer Playground](https://thegraph.com/hosted-service/subgraph/ensdomains/ens)
```

File: docs/web/subgraph.mdx [hunk 2]
```diff
@@ -15,7 +15,7 @@ This is a page covering the graph's ENS subgraph.
 
 ## The Graph
 
-The Graph is protocol for indexing and querying data from blockchains. There are multiple subgraphs that you can use to query information about ENS names.
+The Graph is a protocol for indexing and querying data from blockchains. There are multiple subgraphs that you can use to query information about ENS names.
 These subgraphs are available for [mainnet](https://api.thegraph.com/subgraphs/name/ensdomains/ens) and [goerli](https://api.thegraph.com/subgraphs/name/ensdomains/ensgoerli).
```
Comment (gskril) [inline docs/web/subgraph.mdx:19]: Maybe also add Sepolia while you're here! https://api.studio.thegraph.com/query/49574/enssepolia/version/latest

Comment (TateB) [reply docs/web/subgraph.mdx:19]: and holesky
[https://api.studio.thegraph.com/query/49574/ensholesky/version/latest⁠](https://api.studio.thegraph.com/query/49574/ensholesky/version/latest)

Comment (iamashwin99) [reply docs/web/subgraph.mdx]: Makes sense! Thanks for the links. I have added them in the latest commit.

**Review by lucemans — APPROVED** [review]
lgtm

**Pull request merged**